schema/crdb/dbinit.sql

```diff
@@ -3135,7 +3135,10 @@ CREATE TABLE IF NOT EXISTS omicron.public.blueprint (
     comment TEXT NOT NULL,
 
     -- identifies the latest internal DNS version when blueprint planning began
-    internal_dns_version INT8 NOT NULL
+    internal_dns_version INT8 NOT NULL,
+    -- identifies the latest external DNS version when blueprint planning began
+    -- XXX-dap migration code must set the value for existing blueprints
+    external_dns_version INT8 NOT NULL
```

Comment:
I will obviously fix this before integration but wanted to save it for the last step.  The plan will be to do something similar to internal_dns_version (schema version 36.0.0), setting the external DNS version to 1 for existing blueprints.